packit_service/worker/handler.py

```diff
@@ -190,6 +183,8 @@ def report_rpm_build_failed_because_of_the_srpm_fail(self, build_check_names):
         )
 
     def report_tests_failed_because_of_the_build(self, test_check_names):
+        if not test_check_names:
+            return
```

Comment:
What about having this directly in the `report` method so every `report_*` method can use that?